preprocessing/regex_sentence_segmentation.py

```diff
@@ -0,0 +1,75 @@
+import re
+
+
+def sent_segmentation(text):
+    """Return a list of sentences from the text."""
+    
+    text = text.replace("\n", " ")
+    while "  " in text:
+        text = text.replace("  ", " ")
+
+    sentences = []
+    sent = ""
+
+    for i in range(len(text)-2):
+
+        # sentence can't start with end quotations marks
+        if len(sent) == 0 and text[i] == "\"“":
+            break
+
+        # check if not end of text
+        # check if char is interpunction
+        # check if previous char is not capital (no segmentation on abreviations)
+        # check if char after space is capital
+        # check beginning of next sentence
+        elif (i == len(text)-1 or i != len(text)-3 or i != len(text)-2 or i != 0) \
+            and (re.search(r"[.!?;:]", text[i]) \
```

Comment:
actually, it can be better to make comments to each line instead of one big comment before `if block`